build/documentation/generate-typedocs.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * Utility for generating Typedoc API documentation and placing it in the docs/en/packages folder so it can be hosted by
+ * Guthub pages and viewed through Docusaurus
+ * 
+ * Usage :run node build/documentation/generate-typedocs.js OR
+ * Usage (dry-run): run node build/documentation/generate-typedocs.js --dry-run
+ */
+
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+const os = require('os');
+const { exec } = require('child_process');
+const { spawn } = require("child_process");
+
+const rootDir = path.resolve(process.cwd());
+const srcDir = "packages/*";
+const destDir = path.join("docs", "en", "packages");
+
+var copyReadmeScriptPath = path.join('build', 'documentation', 'copy-package-readme.js');
+var dryRun = false;
+
+const excludedPackages = [
+    "fast-animation",
```

Comment:
Is there a reason we're excluding fast-animation? I think we absolutely want to generate typedocs for the animation package.

Reply:
It was a mistake in the requirements which I just updated.

Reply:
@chrisdholt are there any other packages we should exclude that you can think of?

Reply:
not initially - I think the four below this one make sense to keep undocumented for now.